tests/e2e/Pages/AggregatesComponet.ts

```diff
@@ -73,6 +85,43 @@ class AggregatesComponent extends DashboardPage {
       await expect(layerLocator).toBeVisible();
     }
   }
+
+  async validatesAggregatesInfoButtons() {
+    // click on the first info icon and validate the opopver content
+    await this.aggreagtesTitleInfoIcon.click();
+    const disclaimerText = await this.approximatedisclaimer.textContent();
+    expect(disclaimerText).toContain(
+      'All numbers are approximate and meant to be used as guidance.',
+    );
+
+    // wait for opover layer to be laoded and click to remove it
+    await this.page.waitForTimeout(500);
+    await this.popoverLayer.click();
+
+    // click on the total exposed population info icon and validate the opopver content
+    const exposedPopulationLayer = this.aggregatesLayerRow.filter({
+      hasText: 'Exposed population',
+    });
+    await exposedPopulationLayer.getByTestId('aggregates-info-icon').click();
+    const layerInfoTitle = await this.layerInfoPopoverTitle.textContent();
+    const layerInfoContent = await this.layerInfoPopoverContent.textContent();
+    expect(layerInfoTitle).toContain('Exposed population');
+    expect(layerInfoContent).toContain(
+      'This layer shows the estimated rounded number of people potentially exposed per geographic area.',
+    );
+  }
+
+  async validateLayerPopoverExternalLink() {
+    // Define link to click
+    const layerPopoverExternalLink = this.layerInfoPopoverContent.filter({
+      hasText: 'High Resolution Settlement Layer (HRSL)',
```

Comment:
I believe the sources may be different depending on the country-hazard. We'll deal with the differences for other country-hazard later.

Move the strings to a separate helper file so dealing with the differences in string values is separated from the test logic.

Reply:
Got It! Fixed by mistake in different PR: https://github.com/rodekruis/IBF-system/pull/1658

Reply:
> Got It! Fixed by mistake in different PR: https://github.com/rodekruis/IBF-system/pull/1658

@Piotrk39 Does this mean there's nothing left to review in this PR? If the review comments are resolved else and there's nothing left to review here. Please feel free to merge.

Reply:
Yes indeed. I'll merge